Assets/LeapMotion/Editor/LeapHandPoolEditor.cs

```diff
@@ -0,0 +1,19 @@
+﻿using UnityEngine;
+using UnityEditor;
+using System.Collections;
+
+namespace Leap.Unity {
+  [CustomEditor(typeof(HandPool))]
+  public class LeapHandPoolEditor : Editor {
```

Comment:
We should have this extend CustomEditorBase, which can help reduce errors and also help mimic the custom inspector more reliably.